samples/namespace-monitoring/README.md

```diff
@@ -65,24 +66,24 @@ kubectl label ns $VIYA_ONE_NS sas.com/viya-namespace=$VIYA_ONE_NS
 kubectl label ns $VIYA_TWO_NS sas.com/viya-namespace=$VIYA_TWO_NS
 ```
 
-7. Deploy cluster monitoring (including the Prometheus Operator) with a custom user directory and no SAS Viya dashboards.
+5. Deploy cluster monitoring (including the Prometheus Operator) with a custom user directory and no SAS Viya dashboards.
 ```bash
 VIYA_DASH=false monitoring/bin/deploy_monitoring_cluster.sh
 ```
 
-8. Deploy standard SAS Viya monitoring components each SAS Viya namespace.
+6. Deploy standard SAS Viya monitoring components each SAS Viya namespace.
 ```bash
 VIYA_NS=$VIYA_ONE_NS monitoring/bin/deploy_monitoring_viya.sh
 VIYA_NS=$VIYA_ONE_NS monitoring/bin/deploy_monitoring_viya.sh
 ```
```

Comment:
Shouldn't the env var in the 2nd command should be $VIYA_TWO_NS?

Reply:
Changed